migrations/Version20231005070928.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
```

Comment:
Why do you not use query builder instead of pure SQL?

Reply:
Because:
- it is not available in doctrine migrations
- query builder is used to build queries and is not suitable for building schemas